LabDrivers/utils.py

```diff
@@ -254,6 +265,135 @@ def is_IP_port(device_port, return_vals=False):
     else:
         return True
 
+def list_properties():
+    """
+        This returns a dictionary of all drivers that have properties dictionary in form:
+        return {
+            'NAME': object
+        }
+    """
+    interface = [INTF_VISA, INTF_PROLOGIX, INTF_SERIAL]
+
+    #    instruments.append('TIME')
```

Comment:
One the code runs well, it is best practice to remove the pieces of code which are not used anymore instead of commenting them and adding them to the commit, this make the files to review larger and does not help the reviewer ;)

Reply:
I will look through the code and remove any extraneous code